File: SimpleCalendar.Api/Commands/Events/Event.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace SimpleCalendar.Api.Commands.Events
{
    public class Event
    {
        public string Id { get; set; }

        public string RegionId { get; set; }

        public string Name { get; set; }

        public string Description { get; set; }

        public DateTime StartTime { get; set; }

        public DateTime EndTime { get; set; }

        public DateTime Created { get; set; }

        public string CreatedByEmail { get; set; }

        public DateTime? Published { get; set; }

        public string PublishedByEmail { get; set; }

        public DateTime? Deleted { get; set; }

        public string DeletedByEmail { get; set; }
    }
}

File: SimpleCalendar.App.Schedule/ClientApp/src/services/Api/Models/Event.ts
export interface IBaseEvent {
  id: string
  regionId: string
  name: string
  description: string
}

export interface IEvent extends IBaseEvent {
  startTime: Date
  endTime: Date
}

export interface IEventResponse extends IBaseEvent {
  startTime: string
  endTime: string
}

export interface IEventCreate {
  regionId: string
  startTime: Date
  endTime: Date
  name: string
  description: string
}
